README.md

```diff
@@ -83,7 +83,10 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 #### Text Editors
 
 * [Atom](https://atom.io) - Hackable text editor for the 21st century made by GitHub. [Atom Plugins](editor-plugin.md#atom-plugin). [![Open-Source Software][OSS Icon]](https://github.com/atom/atom) ![Freeware][Freeware Icon]
+* [BBEdit](http://www.barebones.com/products/bbedit/) - The leading professional HTML and text editor for Mac.
+* [Brackets](http://brackets.io) - A modern, open source text editor that understands web design. [![Open-Source Software][OSS Icon]](https://github.com/adobe/brackets/) ![Freeware][Freeware Icon]
```

Comment:
Brackets is an IDE, it should be left in that section.

Edit: Maybe we'll start a web dev IDE section?

Reply:
This is from the official documentation:

> A modern, open source text editor that understands web design.

> With focused visual tools and preprocessor support, Brackets is a modern text editor that makes it easy to design in the browser. It's crafted from the ground up for web designers and front-end developers

Last time when I used it, it was an Atom-like editor, maybe something has changed. Are you sure we want to move back to IDE category ?

Reply:
Maybe a better option is to add a category for basic text editors than can double as IDEs?

If we don't have that already? Will look into it further after work.

Reply:
We can try, the problem is the borders are already blurred (yes emacs/atom are editors but thanks to hyperconfigurability/plugins you can "construct" an "IDE", at the same time, for example,  for Java a configured emacs is still not an IDE) and with another category it can get worse. If have an idea of how we can regroup the editors, let's try it :)